code/game/objects/items/cards_ids.dm

```diff
@@ -117,6 +117,7 @@
 	var/access_txt // mapping aid
 	var/datum/bank_account/registered_account
 	var/obj/machinery/paystand/my_store
+	var/hideaccount = FALSE  // removes account info from examine
```

Comment:
I think this could just be something that indicates that its an electronic ID and all cases of type checking can be removed.